cycode/cli/utils/file_utils.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+
+def change_filename_extension(filename: str, extension: str) -> str:
+    base_name, old_ext = os.path.splitext(filename)
```

Comment:
nice :)

Reply:
it looks like we don't use "old_ext" var. pls rename it to "_"